dataprofiler/profilers/profiler_options.py

```diff
@@ -256,6 +256,10 @@ def __init__(self):
         :ivar histogram_and_quantiles: boolean option to enable/disable
             histogram_and_quantiles
         :vartype histogram_and_quantiles: BooleanOption
+        :ivar num_zeros: boolean option to enable/disable num_zeros
+        :vartype num_zeros: BooleanOption
+        :ivar num_negatives: boolean option to enable/disable num_negatives
+        :vartype num_negatives: BooleanOption
```

Comment:
You should add tests for these things not just in `test_profiler_options.py` but also `test_numerical_options.py` since these are found in `NumericalOptions` there you should add cases for your 2 new options anywhere new logic for them was added (e.g. `validate_helper`).